whois-update/src/test/java/net/ripe/db/whois/update/authentication/credential/PgpCredentialValidatorTest.java

```diff
@@ -35,13 +45,16 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class PgpCredentialValidatorTest {
-    @Mock private PreparedUpdate update;
+    @Mock private PreparedUpdate preparedUpdate;
+
     @Mock private UpdateContext updateContext;
     @Mock private RpslObjectDao rpslObjectDao;
     @Mock private DateTimeProvider dateTimeProvider;
     @Mock private LoggerContext loggerContext;
     @InjectMocks private PgpCredentialValidator subject;
 
+    private final Update update = getUpdate();
```

Comment:
final Update instance is shared between tests, but contains mutable state. Better to remove final and create in Before method instead